### PR TITLE
Removes abiotic check for sleepers, cryo, body scanner, and DNA scanner.

### PR DESCRIPTION
**Pull request by 9600bauds**
Apparently the only reason why we even had this was that long, long, LONG ago, people could drop things inside sleepers and those things would get stuck inside the sleeper's contents, and no coder was smart enough to write a for check to spit out the bad contents.

Anyone that has played Medical can attest to the abiotic check being a pain when you're trying to scan the miner who broke his hand and you have to disarm his other hand just to put him inside the body scanner.

<Ririchiyo> if you guys actually want people to work on that kind of stuff people need to stop mucking around with files related to it

### CODE REVIEW

File: code/game/dna/dna_modifier.dm [hunk 1]
```diff
@@ -111,15 +111,10 @@
 
 /obj/machinery/dna_scannernew/proc/eject_occupant(var/exit = loc)
 	src.go_out(exit)
-	for(var/obj/O in src)
-		if(!istype(O,/obj/item/weapon/circuitboard/clonescanner) && \
-		   !istype(O,/obj/item/weapon/stock_parts) && \
-		   !istype(O,/obj/item/stack/cable_coil) && \
-		   O != beaker)
-			O.loc = get_turf(src)//Ejects items that manage to get in there (exluding the components and beaker)
```
Comment (N3X15) [inline code/game/dna/dna_modifier.dm]: Why was this removed?  It's supposed to eject shit that gets dropped inside.

**Comment by 9600bauds** [comment]
you've been visited by the EGGMAN of COLLABORATOR STATUS

fast merges and "needs final decision" will come to you, but ONLY if you comment "ty eggman of coding"

**Comment by Exxion** [comment]
Storage atoms when
